src/configLoader.js

```diff
@@ -1,3 +1,4 @@
+import changeCase from 'change-case';
```

Comment:
I think this can go now along with the addition in package.json/package-lock.json